Procfile.production

```diff
@@ -0,0 +1,9 @@
+# Basic procfile for rails-related part of dev processes
+# using a webpack development server to load JavaScript and CSS
+
+# Development rails requires both rails and rails-assets
+# (and rails-server-assets if server rendering)
+rails: RAILS_ENV=production rails s -b 0.0.0.0
+
+# Run the hot reload server for client development
```

Comment:
If you think that we need these comments in Procfiles - leave it, if not - please remove